Test failure is unrelated

```
FAILED tests/components/config/test_automation.py::test_update_automation_config_with_error[automation_config0] - assert {'message': "...ta['action']"} == {'message': "...a['trigger']"}
  Differing items:
  {'message': "Message malformed: required key not provided @ data['action']"} != {'message': "Message malformed: required key not provided @ data['trigger']"}
  Full diff:
  - {'message': "Message malformed: required key not provided @ data['trigger']"}
  ?                                                                    - ^^^^
  + {'message': "Message malformed: required key not provided @ data['action']"}
  ?                                                                   ++  ^^
Error: Process completed with exit code 1.
```

--- a/homeassistant/components/plex/manifest.json
+++ b/homeassistant/components/plex/manifest.json
@@ -4,7 +4,7 @@
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/plex",
   "requirements": [
-    "plexapi==4.13.1",
+    "plexapi==4.13.2",
     "plexauth==0.0.6",
     "plexwebsocket==0.0.13"
   ],

--- a/requirements_all.txt
+++ b/requirements_all.txt
@@ -1333,7 +1333,7 @@ pillow==9.3.0
 pizzapi==0.0.3
 
 # homeassistant.components.plex
-plexapi==4.13.1
+plexapi==4.13.2
 
 # homeassistant.components.plex
 plexauth==0.0.6

--- a/requirements_test_all.txt
+++ b/requirements_test_all.txt
@@ -960,7 +960,7 @@ pilight==0.1.1
 pillow==9.3.0
 
 # homeassistant.components.plex
-plexapi==4.13.1
+plexapi==4.13.2
 
 # homeassistant.components.plex
 plexauth==0.0.6

